summaries/search-based-survey.md

```diff
@@ -0,0 +1,335 @@
+# Search-Based Software Test Data Generation
+**Article:**
+*Phil McMinn*. Search-Based Software Test Data Generation: A Survey. Software Testing, Verification and Reliability, vol. 14, no. 2, pp. 105–156, 2004.
+
+## Metaheursitic Search Applications
+**Goals:**
+ - Coverage of the code.
+ - Exercise a specific feature of the program.
+ - Disprove grey-box properties.
+ - Verify non-functional properties.
+
+## Metaheursitic Search Techniques
+Solution should be encoded such that they can be manipulated by the search and 'neighbours' can be compared. 
+Using an *objective function*, the search distinguishes *good* solutions from *bad* solutions and guides the search using this function.
```

Comment:
I read it as "the search guides the search", but I think you mean the mean "the objective function guides the search"?